doc/api/cli.md

```diff
@@ -76,6 +76,21 @@ $ node --completion-bash > node_bash_completion
 $ source node_bash_completion
 ```
 
+### `-u`, `--conditions=conditionA,conditionB`
```

Comment:
Our options parser natively supports passing an option multiple times, e.g. `-u condA -u condB` or `--conditions condA --conditions=condB`. Using that tends to be a bit nicer, because it means that one piece of code (in a tool that spawns a Node.js process) can set a flag without worrying about whether another piece of code has set the same flag already.

Specifying the type of the option variable as `std::vector<std::string>` instead of a plain `std::string` is sufficient for that.

(In that case it might make sense to name the option `--condition`, i.e. singular)

Reply:
Thanks for mentioning this - I've gone ahead and added that and it is certainly preferable to comma-separated.

I've kept it as a plural form though for now as it still doesn't sound odd to me gramatically using the plural for this.